dipper/sources/ClinVarXML_alpha.py

```diff
@@ -361,12 +361,16 @@ def allele_to_triples(allele, triples) -> None:
     if allele.label is not None:
         write_spo(allele.id, 'rdfs:label', allele.label, triples)
 
-    # <ClinVarVariant:rcv_variant_id><OWL:sameAs><dbSNP:rs>
+    # <ClinVarVariant:rcv_variant_id><OWL:hasDbXref><dbSNP:rs>
+    #
+    # Note that making clinvar variants and dbSNPs equivalent
+    # causes clique merge bugs, so best to leave them as xrefs
+    # Example: https://www.ncbi.nlm.nih.gov/clinvar/variation/31915/
+    # https://www.ncbi.nlm.nih.gov/clinvar/variation/21303/
     for dbsnp_id in allele.dbsnps:
-        # sameAs or hasdbxref?
         write_spo(
             allele.id,
-            'owl:sameAs',
+            'oboInOwl:hasDbXref',
```

Comment:
`self.globaltt['database_cross_reference']`

@kshefchek  always use the global mapping